### PR TITLE
Octopus Pipe for Bitbucket: octopus-cli-run

### PR DESCRIPTION
**Pull request by harrisonmeister**
New blog post which covers an overview of the new _experimental_ Bitbucket Pipe for Octopus, `octopus-cli-run`

### CODE REVIEW

File: blog/2020-05/octopus-bitbucket-pipe/index.md [hunk 1]
```diff
@@ -0,0 +1,258 @@
+---
+title: "Octopus Pipe for Bitbucket: octo"
+description: An overview of the new experimental Bitbucket Pipe for Octopus, octo.
```
Comment (robpearson) [inline blog/2020-05/octopus-bitbucket-pipe/index.md:3]: Minor point but can I request that you structure this like `Learn how to integrate Octopus and BitBucket with our new experimental BitBucket Pipe called octo.` or something like that.

File: blog/2020-05/octopus-bitbucket-pipe/index.md [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+author: mark.harrison@octopus.com
+visibility: private
+published: 2020-05-30
+metaImage: bitbucket-cd.png
+bannerImage: bitbucket-cd.png
+tags:
+ - DevOps
+---
+
+![Bitbucket Pipelines](bitbucket-cd.png)
```
Comment (robpearson) [inline blog/2020-05/octopus-bitbucket-pipe/index.md:13]: Love the intro. Great work!

File: blog/2020-05/octopus-bitbucket-pipe/index.md (hunk 1 above)
Comment (harrisonmeister) [reply blog/2020-05/octopus-bitbucket-pipe/index.md]: Sounds good to me, have changed the description :+1:

**Comment by harrisonmeister** [comment]
Thanks for the feedback @robpearson - I did wonder about calling out the experimental more prominently. For example, a heading just after the `In this post` section - a bit like in the Tomcat post - https://octopus.com/blog/ultimate-guide-to-tomcat-deployments#a-note-on-the-postgresql-server

I'll wait to see what @wordlee thinks, and can add if needed :smile:

**Comment by wordlee** [comment]
> Thanks for the feedback @robpearson - I did wonder about calling out the experimental more prominently. For example, a heading just after the `In this post` section - a bit like in the Tomcat post - https://octopus.com/blog/ultimate-guide-to-tomcat-deployments#a-note-on-the-postgresql-server
> 
> I'll wait to see what @wordlee thinks, but I could add something like this:
> 
> > :::warning
> > **Experimental Pipe:**
> > The `octo` Bitbucket Pipe is currently experimental.
> > If you want to try out the latest integration, and only need to use some of the more commonly used commands, for example to manage your packages, releases and deployments - then using the Pipe will be the right choice for you.
> > However, If you need full control over integrating your Bitbucket Pipeline with Octopus, then the pre-configured CLI Docker image would be the recommended method to do that.
> > :::

I think what you have in the post is fine, I'd probably just extend the opening line to say something like `using the experimental pipe isn't supported yet, but if you're interested in trying it out, read on`. Or something similar. I can play around with verbiage when I review the post (next week now).

**Comment by wordlee** [comment]
An aside from the content of the blog. If at all possible, it might be worth rethinking the pipe name. octo was the name for what is now the Octopus CLI. In context it's unlikely anybody will get confused, but we should avoid giving too many things the same name :)

**Comment by harrisonmeister** [comment]
> An aside from the content of the blog. If at all possible, it might be worth rethinking the pipe name. octo was the name for what is now the Octopus CLI. In context it's unlikely anybody will get confused, but we should avoid giving too many things the same name :)

Thanks @wordlee - I'm happy to rename it,  but it'd make sense to do that before we publish the post.if we are going to rename.it

@robpearson - thoughts on a rename or who to get feedback from?

**Pull request merged**